htdocs/compta/facture/card.php

```diff
@@ -3692,7 +3693,7 @@ function setRadioForTypeOfIncoice() {
 	}
 
 	// Payment term
-	print '<tr><td class="nowrap fieldrequired">'.$langs->trans('PaymentConditionsShort').'</td><td colspan="2">';
+	print '<tr><td class="nowrap ">'.$langs->trans('PaymentConditionsShort').'</td><td colspan="2">';
```

Comment:
Looks strange.
Payment term is a mandatory information on a invoice (in legal point of view).

Reply:
functionally I agree with you on the mandatory aspect of this field but it is not mandatory in terms of the table constraints, hence my proposal to remove the indicators